src/beaminteraction/src/4C_beaminteraction_submodel_evaluator_beamcontact.cpp

```diff
@@ -244,6 +244,29 @@ void BeamInteraction::SubmodelEvaluator::BeamContact::setup()
   issetup_ = true;
 }
 
+/*----------------------------------------------------------------------*
+ *----------------------------------------------------------------------*/
+bool BeamInteraction::SubmodelEvaluator::BeamContact::have_lagrange_dofs() const
+{
+  if (assembly_managers_.size() > 0)
```

Comment:
```suggestion
  if (assembly_managers_.empty()) return false;
```

Minor: use early return.